projekt_do_pracy_na_labach/lab04/backend-deployment.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: backend-deployment
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: backend
+  template:
+    metadata:
+      labels:
+        app: backend
+    spec:
+      initContainers:
+      - name: init-db
+        image: busybox
+        command: ["sh", "-c", "if [ -e /dsb/backup/example.txt ]; then echo 'Plik istnieje'; cp -r /dsb/backup/* /data/db/; else echo 'Plik nie istnieje'; fi"]
```

Comment:
Jak coś to generalnie zadanie jest jako tako dobrze, ale fajnie by było zrobić tego busy box'a w statefullSecie abazy danych a nie w backendzie :p